src/test/java/seedu/realodex/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -104,6 +105,15 @@ public EditPersonDescriptorBuilder withTags(String... tags) {
         return this;
     }
 
+    /**
+     * Sets the {@code Housing type} of the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withHousingType(String housingType) {
+        HousingType h = new HousingType(housingType);
```

Comment:
```suggestion
        HousingType housingType = new HousingType(housingType);
```

careful about variable naming

Reply:
I think its fine for that particular class, because iirc all variables are named that same way

Reply:
naw its becasue the input string is housingType already

the below method does the same thing

since its a very short time to use variable it should be fine?